.appveyor.yml

```diff
@@ -62,7 +62,10 @@ install:
 # Not a project with an msbuild file, build done at install.
 build: None
 
+# Makefiles support provided by MinGW, which does POSIX translations, so run tests natively as well
 test_script:
+  - cmd: make clean
+  - cmd: py.test
```

Comment:
Yes, but because make is run through MinGW, there are some POSIX translations that are not representative of running the code directly on Windows. Say somebody submits a PR with hardcoded slashes instead of using `os.path.join()`, tests (or the application) might pass with `make clean` but fail when running py.test.